yt_dlp/extractor/skeb.py

```diff
@@ -1,140 +1,117 @@
-                    'ext': 'vtt',
-                }],
-            },
-            'duration': 98,
             'ext': 'mp3',
-            'vcodec': 'none',
-            'abr': 128,
+            'title': '3',
+            'description': 'md5:6de1f8f876426a6ac321c123848176a8',
+            'display_id': '3',
+            'duration': 98,
+            'genres': ['voice'],
+            'tags': 'count:11',
+            'thumbnail': r're:https?://.+',
+            'uploader': '古川ノブ@宮城の動画勢Vtuber',
+            'uploader_id': 'furukawa_nob',
         },
     }, {
-        'url': 'https://skeb.jp/@mollowmollow/works/6',
+        'url': 'https://skeb.jp/@Rizu_panda_cube/works/626',
         'info_dict': {
-            'id': '6',
-            'title': 'ヒロ。\n\n私のキャラク... by 諸々',
-            'description': 'md5:aa6cbf2ba320b50bce219632de195f07',
-            '_type': 'playlist',
-            'entries': [{
-                'id': '486430',
-                'title': 'ヒロ。\n\n私のキャラク... by 諸々',
-                'description': 'md5:aa6cbf2ba320b50bce219632de195f07',
-            }, {
-                'id': '486431',
-                'title': 'ヒロ。\n\n私のキャラク... by 諸々',
-            }],
+            'id': '626',
         },
+        'playlist_count': 2,
     }]
 
+    def _call_api(self, uploader_id, work_id):
+        return self._download_json(
+            f'https://skeb.jp/api/users/{uploader_id}/works/{work_id}', work_id, headers={
+                'Accept': 'application/json',
+                'Authorization': 'Bearer null',
+                'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/138.0.0.0 Safari/537.36',
+            })
+
     def _real_extract(self, url):
-        video_id = self._match_id(url)
-        nuxt_data = self._search_nuxt_data(self._download_webpage(url, video_id), video_id)
+        uploader_id, work_id = self._match_valid_url(url).group('uploader_id', 'id')
+        try:
+            works = self._call_api(uploader_id, work_id)
+        except ExtractorError as e:
+            if not isinstance(e.cause, HTTPError) or e.cause.status != 429:
+                raise
+            webpage = e.cause.response.read().decode()
+            value = self._search_regex(
+                r'document\.cookie\s*=\s*["\']request_key=([^;"\']+)', webpage, 'request key')
+            self._set_cookie('skeb.jp', 'request_key', value)
+            works = self._call_api(uploader_id, work_id)
 
-        parent = {
-            'id': video_id,
-            'title': nuxt_data.get('title'),
-            'description': nuxt_data.get('description'),
-            'uploader': traverse_obj(nuxt_data, ('creator', 'name')),
-            'uploader_id': traverse_obj(nuxt_data, ('creator', 'screen_name')),
-            'age_limit': 18 if nuxt_data.get('nsfw') else 0,
-            'tags': nuxt_data.get('tag_list'),
+        info = {
+            'title': work_id,
+            'display_id': work_id,
```

Comment:
I don't like duplicating the playlist id as the entries' `title` and `display_id`

`display_id` is redundant if it's the same as the playlist id, could be removed

if there's really no alternative we could use for the entries' `title`, then let's enumerate the `previews` and use `{work_id}-{idx}`